src/core/svgs/stories.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react"
+import { css } from "@emotion/core"
+import { ThemeProvider } from "emotion-theming"
+
+import { storybookBackgrounds } from "@guardian/src-helpers"
+import { brand } from "@guardian/src-foundations/themes"
+
+import { iconDefault, iconSmall } from "../components/button/styles"
```

Comment:
Could we inline the relevant styles here? It's a pain having cross-package references